cityofaustin/formstack-alpha-theme
# 3903 Form question runs over iFrame margin

Zenhub issue: https://app.zenhub.com/workspaces/techstack-5a78b88e1ce69f3510b678ef/issues/cityofaustin/techstack/3903

This fixes the margin overflow issue with Edge!

To Test: 
- See: https://cityofaustin.formstack.com/forms/bob_s_rando
- Use browser stack to view on Edge(latest)
- Do the radio button header text overflow and spill beyond the form container? 🚫? 
- ... Or, break the text into multi lines within the container ✅

What fixed it? 
- MS Edge dosen't give no respect to `<legend>` tags. 
- Adding `  width: 100%;` to all legend tags forces it to *RESPECT* parent's width constraints!
